uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/instances/leave-request-instance_test.js

```diff
@@ -410,5 +410,131 @@ define([
         });
       });
     });
+
+    describe('check status methods', function () {
+
+      var optionGroupDeferred,
+        mockOptionValue,
+        promise;
+
+      function commonSetup(statusName, methodName) {
+        optionGroupDeferred = $q.defer();
+        mockOptionValue = [{
```

Comment:
There is no reason for you to mock the data here when you have already mocked data in test/mock/data